spec/jobs/spree_klaviyo/analytics_event_job_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe SpreeKlaviyo::AnalyticsEventJob do
+  include ActiveJob::TestHelper
+
+  let(:event_name) { 'test_event' }
+  let(:customer_properties) { { email: 'test@example.com', guest_id: 'visitor_123' } }
+  let(:event_properties) { { resource: 'test_resource' } }
+  let(:time) { Time.current }
+  let(:klaviyo_integration) { double('KlaviyoIntegration') }
+  let(:store) { double('Store') }
+
+  before do
+    allow(subject).to receive(:store).and_return(store)
+    allow(store).to receive(:integrations).and_return(double('Integrations'))
+    allow(store.integrations).to receive(:active).and_return(double('ActiveIntegrations'))
+    allow(store.integrations.active).to receive(:find_by).with(type: 'Spree::Integrations::Klaviyo').and_return(klaviyo_integration)
+  end
```

Comment:
@jawaria-sparksol please don't use stubbing like this. Just create the integration via factory